cpp/lib/src/Zeder.cc

```diff
@@ -620,17 +620,70 @@ void FullDumpDownloader::parseRows(const Params &params, const std::shared_ptr<J
 }
 
 
-bool FullDumpDownloader::download(EntryCollection * const collection) {
+bool FullDumpDownloader::download(EntryCollection * const collection, bool disable_cache_mechanism) {
     const auto params(dynamic_cast<FullDumpDownloader::Params * const>(downloader_params_.get()));
     if (unlikely(params == nullptr))
         LOG_ERROR("dynamic_cast failed!");
 
     std::unordered_map<std::string, ColumnMetadata> column_to_metadata_map;
     std::shared_ptr<JSON::JSONNode> json_data;
+    std::shared_ptr<JSON::JSONNode> json_cached_data;
+
+    bool use_cache = not disable_cache_mechanism;
+    bool cache_present = false;
+    bool downloaded = false;
+    std::string zeder_cache_path = UBTools::GetTuelibPath();
+    if (StringUtil::EndsWith(params->endpoint_url_, "ixtheo", true))
```

Comment:
Another option would be to replace the endpoint_path by the Zeder Flavour in the params class. Then the downloader could determine the url internally and still have the information about the instance in a more structured way. What do you think?